### PR TITLE
Issue 17

### PR DESCRIPTION
**Pull request by korikuzma**

### CODE REVIEW

File: gene/database.py [hunk 1]
```diff
@@ -159,3 +185,127 @@ def create_meta_data_table(self, existing_tables: List[str]):
...
+            return None
+        except (KeyError, IndexError):  # record doesn't exist
+            return None
+
+    def get_records_by_type(self, query: str,
+                            match_type: str) -> List[Dict]:
+        """Retrieve records for given query and match type.
+        :param query: string to match against
+        :param str match_type: type of match to look for. Should be one
+            of {"label", "alias", "xref", "associated_with"} (use
```
Comment (jsstevenson) [inline gene/database.py:229]: is "symbol" also an acceptable match type for `get_records_by_type`?

Comment (korikuzma) [reply gene/database.py]: Oops. Didn't update docs. It should be symbol, prev_symbol, alias, xref, and associated_with!

Comment (korikuzma) [reply gene/database.py]: Oops. Didn't update docs. It should be symbol, prev_symbol, alias, xref, and associated_with!

File: gene/schemas.py [hunk 2]
```diff
@@ -507,3 +636,131 @@ def schema_extra(schema: Dict[str, Any],
                     'url': 'https://github.com/cancervariants/gene-normalization'  # noqa: E501
                 }
             }
+
+
+class NormalizeService(BaseModel):
+    """Define model for returning normalized concept."""
+
+    query: str
+    warnings: Optional[List]
```
Comment (jsstevenson) [inline gene/schemas.py:645]: does the List always contain Dicts or can it contain strings too?

**Review by jsstevenson — APPROVED** [review]

File: gene/schemas.py (hunk 2 above)
Comment (jsstevenson) [reply gene/schemas.py:645]: (I think this would apply to the other service model as well?)

Comment (korikuzma) [reply gene/schemas.py:645]: Good question. I think right now it's just Dicts

**Review by jsstevenson — APPROVED** [review]

**Pull request merged**